project/AppDependencies.scala

```diff
@@ -1,33 +1,31 @@
-import play.core.PlayVersion
-import sbt._
+import sbt.*
 
 object AppDependencies {
 
-  val bootstrapVersion = "7.15.0"
+  val bootstrapVersion = "7.21.0"
+  val mongoVersion = "1.3.0"
 
   private lazy val compile = Seq(
     play.sbt.PlayImport.ws,
-    "uk.gov.hmrc.mongo"       %% "hmrc-mongo-play-28"             % "0.74.0",
-    "uk.gov.hmrc"             %% "play-frontend-hmrc"             % "6.2.0-play-28",
-    "uk.gov.hmrc"             %% "domain"                         % "8.2.0-play-28",
+    "uk.gov.hmrc.mongo"       %% "hmrc-mongo-play-28"             % mongoVersion,
+    "uk.gov.hmrc"             %% "play-frontend-hmrc"             % "7.17.0-play-28",
```

Comment:
The acceptance tests are failing [here](https://github.com/hmrc/trusts-acceptance-tests/blob/54d2f411b2d69f16765af71bcc2def624dcf6946/src/test/resources/features/non_taxable_to_taxable/assets/Assets.feature#L86) due to a change made in [version 6.6.0](https://github.com/hmrc/play-frontend-hmrc/blob/main/CHANGELOG.md#changed-20).

I think the [InputTextCharacterCount.scala.html](https://github.com/hmrc/maintain-trust-assets-frontend/blob/DDCE-4453/app/views/components/InputTextCharacterCount.scala.html#L59) component needs to be updated to use `govuk-character-count`

Reply:
oh I see I had run the run_maintain and that doesnt include maintainAssets tag this why, yeah I know the fix as I fixed it in other jiras, pushed the fix https://github.com/hmrc/trusts-acceptance-tests/pull/772